src/extensions/default/BrambleUrlCodeHints/style.css

```diff
@@ -106,3 +106,956 @@
 #selfie-video-bg.on {
     opacity: 1;
 }
+
+#filter-list {
```

Comment:
Some options here might be http://camanjs.com, http://fabricjs.com/image-filters and https://www.viget.com/articles/instagram-style-filters-in-html5-canvas, although there are no doubt a number of other solutions available out there.